web/concrete/src/Asset/CssAsset.php

```diff
@@ -143,8 +143,14 @@ protected static function process($assets, $processFunction)
             if (!file_exists($cacheFile)) {
                 $css = '';
                 foreach ($assets as $asset) {
-                    if ($asset->getAssetPath()) {
-                        $css .= file_get_contents($asset->getAssetPath()) . "\n\n";
+                    $contents = false;
+                    if ($asset->assetIsLocaleDependent()) {
+                        //@todo
```

Comment:
@aembler Which is the best way to "execute the default route function and returning the content" ?